### PR TITLE
Add `docker_sets['setfile']` parameter to travis.yml

### PR DESCRIPTION
**Pull request by mcanevet**

### CODE REVIEW

File: config_defaults.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@
     dist: trusty
     services: docker
     bundler_args: --with system_tests
-    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=@@COLLECTION@@ BEAKER_set=@@SET@@ BEAKER_TESTMODE=@@TESTMODE@@
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=@@COLLECTION@@ BEAKER_set=@@SET@@ BEAKER_setfile=@@SETFILE@@ BEAKER_TESTMODE=@@TESTMODE@@
```
Comment (DavidS) [inline config_defaults.yml]: beaker-rspec does the right thing here:

https://github.com/puppetlabs/beaker-rspec/blob/da9259ffab8fdb248b3fa5c2e808f94a3b71625a/lib/beaker-rspec/spec_helper.rb#L41

Comment (mcanevet) [reply config_defaults.yml]: @DavidS what am I supposed to set in my `.sync.yml` then?
I tried with:
```yaml
.travis.yml:
  docker_sets:
    - set: 'centos7-64{hypervisor=docker}'
```
But it's looking for a file named `spec/acceptance/nodeset/centos7-64{hypervisor=docker}.yaml`

**Comment by DavidS** [comment]
@TheMeier work to support centos8 across Puppet, Inc. is currently under way.

For the modules we (the content team) are supporting, we're moving over to litmus wholesale, as it allows us to re-use content from across the product (e.g. the puppet_agent::install task) and surface full PR acceptance testing on travis for our open source module.

**Comment by scotje** [comment]
@rodjek @glennsarti thoughts on this?

**Comment by glennsarti** [comment]
There's an outstanding question for @DavidS

File: README.md [hunk 2]
```diff
@@ -85,7 +85,8 @@ Travis uses a .travis.yml file in the root of your repository to learn about you
...
 | global\_env     |Allows you to set global environment variables which will be defined for all travis jobs; for example, `PARALLEL_TEST_PROCESSORS` or `TIMEOUT`.  See the [Travis Global Environment Variables](https://docs.travis-ci.com/user/environment-variables/#Global-Variables) documentation for details.|
 |docker\_sets     |Allows you to configure sets of docker to run your tests on. For example, if I wanted to run on a docker instance of Ubuntu I would add  `set:docker/ubuntu-14.04` to my docker\_sets attribute.  The docker_sets is a hash that supports the 'set', 'testmode', and 'collection' keys. |
-|docker\_sets['set']| This should refrence the docker nodeset that you wish to run. |
+|docker\_sets['set']| This should reference the docker nodeset that you wish to run. |
+|docker\_sets['setfile']| This should reference the docker nodeset generated by beaker-hostgenerator that you wish to run. |
```
Comment (glennsarti) [inline README.md]: nit: This should probable be

`This should reference the docker nodeset file generated by beaker-hostgenerator that you wish to run.`  Added the word file.

**Comment by mcanevet** [comment]
Actually I don't need this anymore as I moved to litmus.
I'll close this PR.

**Pull request closed without merging**